### DISCUSSION TITLE
minimum transaction amount

### DISCUSSION THREAD

**Discussion by persisty** [dogecoin/dogecoin#2731] [Help needed] [answered]
Hi,
How can one change the minimum transaction amount on a node?
It seems to be 1 Doge by default.

I tried 0.9 Doge and it gives:

error code: -4
error message:
Transaction amount too small

Thanks very much in advance

**Comment by patricklodder** [comment, marked as answer]
You can set this with `-discardthreshold=<amount>` on the command line or in dogecoin.conf:

```
  -discardthreshold=<amt>
       The minimum transaction output size (in DOGE) used to validate wallet
       transactions and discard change (to fee) (default: 1.00)
```

Note that you risk that your transaction will not be mined setting this under 1 DOGE, and you should not set this lower than 0.01, as that is what most nodes enforce as the dust limit.

**Reply by persisty** [reply]
Thank you. Got it.
